src/algebra/order/lattice_group.lean

```diff
@@ -72,17 +72,28 @@ variables {α : Type u} [lattice α] [comm_group α]
 @[to_additive]
 lemma mul_sup [covariant_class α α (*) (≤)] (a b c : α) : c * (a ⊔ b) = (c * a) ⊔ (c * b) :=
 begin
-  refine le_antisymm _ (by simp),
+  refine le_antisymm _ _,
   rw [← mul_le_mul_iff_left (c⁻¹), ← mul_assoc, inv_mul_self, one_mul],
-  exact sup_le (by simp) (by simp),
+  apply sup_le,
+  { rw le_inv_mul_iff_mul_le, exact le_sup_left, },
+  { rw le_inv_mul_iff_mul_le, exact le_sup_right },
+  { rw [sup_le_iff, mul_le_mul_iff_left, mul_le_mul_iff_left, and_iff_left le_sup_right],
+    exact le_sup_left, },
 end
 
 @[to_additive]
 lemma mul_inf [covariant_class α α (*) (≤)] (a b c : α) : c * (a ⊓ b) = (c * a) ⊓ (c * b) :=
 begin
-  refine le_antisymm (by simp) _,
+  refine le_antisymm _ _,
+  { rw [le_inf_iff, mul_le_mul_iff_left, mul_le_mul_iff_left,
+    and_iff_left inf_le_right],
+    exact inf_le_left, },
   rw [← mul_le_mul_iff_left (c⁻¹), ← mul_assoc, inv_mul_self, one_mul],
```

Comment:
I'd lean towards surrounding this line till the end of the proof in braces, to keep things symmetric.

Reply:
Done.